HelpSource/Classes/MIDIClient.schelp

```diff
@@ -78,10 +78,37 @@ On non-linux systems, it posts a warning and returns nil.
 
 EXAMPLES::
 
+This is Linux specific code.
+The function takes care of initializing MIDIClient, detects a MIDI device named SE25 and connects it to port 0 of SuperCollider.
+
 code::
-MIDIClient.init;
+(
+~connect_midi_devices = { 
+	// init or refresh MIDIClient end points
+	if(MIDIClient.initialized == false) { 
+		MIDIClient.init; 
+	} {
+		MIDIClient.list; 
+	};
+
+	MIDIClient.sources.do({ arg endPoint;
+		if(endPoint.device == "SE25" and: { endPoint.name == "SE25 MIDI 1" }) {
+			// catch exception thrown when already connected
```

Comment:
Can't the device / name be specified as arguments to the MIDIClient ~connect_midi_device function? That way, we show a more generalized use case that can applied anywhere this example may be run and it gives users a more flexible code sample that they can copy / paste / use

Reply:
I pushed a commit to address this